The Variant Effect Predictor (VEP) community tool predicts the impact of non-synonymous single amino acid variants on any protein. Any isoform can be selected provided it aligns perfectly with an ENSP. By zooming in on a region of the sequence, one or more variants can be selected by simply clicking on a residue in the sequence and providing the variant amino acid, followed by clicking on the Get Predictions button. The Add Multiple Variants button allows one to select multiple variants in one go by entering their position in the sequence and variant amino acid, and obtain their predicted effect by clicking on the Get Predictions button. It is also possible to upload a CSV file containing the position in the sequence, the original and variant amino acid for each variant and get their predicted impact. The SIFT ([Kumar, Henikoff and Ng, 2009](https://doi.org/10.1038/nprot.2009.86)) and PolyPhen-2 ([Adzhubei _et al._, 2010](https://doi.org/10.1038/nmeth0410-248)) scores for the selected variant(s) are obtained from the Ensembl VEP tool ([McLaren _et al._, 2016](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0974-4)) and returned in a table, which can be exported in CSV format. While these _in silico_ predictions are not a substitute for the experimental determination of the effect of a variant, they can be combined with the information in the Medical, Sequence and Phenotype views to provide insight into the impact of the variant.
