Developed in collaboration between the Swiss Institute of Bioinformatics (SIB) and GeneBio, neXtProt is a human-centric discovery platform offering its users a seamless integration of and navigation through proteins-related data. It is expected to become the central resource for any life scientist that requires high-quality protein-centric information in the context of modern biomedical research. The development of neXtProt started in April 2009 and a tremendous amount of work has already been done. neXtProt already contains a vast amount of information obtained by mining many external data resources with very stringent quality criteria. The January 2011 version of neXtProt Beta encompasses the following:

* All the data encapsulated in human protein entries in Swiss-Prot;
* Mapping of proteins to their genomic transcripts on the human genome using Ensembl;
* Protein expression information from antibodies obtained from the Human Proteomics Atlas (HPA) developed by the group of Mathias Uhlen at KTH Royal Institute of Technology in Stockholm;
* Micro-array and cDNA expression information from the Bgee resource developed by the SIB Evolutionary Bioinformatics group;
* Additional single-amino acid polymorphisms (SAPs) variants obtained from Ensembl;
* Additional identifiers such as cDNA clone names encoding for the proteins, Affymetrix and Illumina DNA probesets;
* When possible, disease-related controlled vocabularies mapped to MESH;
* Abstracts of all articles from PubMed that are cited in human Swiss-Prot entries;
* GO annotations records as well as all those computationally mapped to human entries by the UniProt consortium;
* For every integrated data, a GOLD or SILVER quality tag is provided;
* External links to more than 80 external resources.

This rich, high-quality content is made available via an innovative web interface that, amongst many other functionalities, currently features the following:

* An intuitive Google-like search enabling both simple and relatively complex queries. These searches are rendered more precise using dedicated topics;
* An intuitive display of search results to allows easy selection of proteins of interest;
* An original way of visualising proteins entries: they can be seen from three different perspectives: the “Protein”, the underlying “Gene” and the “References” used to annotate it;
* Each of those is further subdivided in views (function, medical, expression, exons, curated publications, etc.) that put the available information in context;
* A sequence features viewer that seamlessly links a protein sequence to graphical and tabular representations of its annotations;
* Dedicated pages for publications featuring, for each publication, the proteins that were annotated with the publication under scrutiny;
* Dedicated pages for controlled vocabulary terms with links to related proteins as well as graphical and tree representations of ontologies;
* Free and certified https secured access to neXtProt for logged-in users;
* For logged-in users, the capability to personalize their usage of neXtProt by favoring, and tagging the searches, proteins or publications of their choice.
